### ISSUE TITLE
about  loss function

### ISSUE THREAD

**Issue by dmuqlzhang** [2014gaokao/pedestrian-attribute-recognition-with-GCN#3]
excuse me,  I   want to understand why not add label weight    at  Multilabelloss  function? I think maybe  use weighted   cross entropy loss    will achieve a better result？

**Comment by abhigoku10** [comment]
@dmuqlzhang  did u try using wce loss did u get better accuracy ?
